DSpace/dspace-angular
# Add special groups list to the profile page

## References
_Add references/links to any related issues or PRs. These may include:_
* Fixes [#8206](https://github.com/DSpace/DSpace/issues/8206)
* Requires DSpace/DSpace#8276

## Description
This PR add the list of special groups, if any, who the user belong to in the profile page
![Schermata da 2022-04-29 20-51-05](https://user-images.githubusercontent.com/2486489/166008072-ea767968-b6a9-4507-9950-15a67459e9d4.png)

## Instructions for Reviewers
To test this PR : 
- Using the rest branch mentioned run the rest server locally and create a new `Submitters` group
- in your local.cfg add this property and restart your server
```
authentication-password.login.specialgroup = Submitters
```
- after login in the profile page shoould appear `Submitters` in the special group list

**Include guidance for how to test or review your PR.** This may include: steps to reproduce a bug, screenshots or description of a new feature, or reasons behind specific changes. 

## Checklist
_This checklist provides a reminder of what we are going to look for when reviewing your PR. You need not complete this checklist prior to creating your PR (draft PRs are always welcome). If you are unsure about an item in the checklist, don't hesitate to ask. We're here to help!_
